doc/whats_new/v1.4.rst

```diff
@@ -36,6 +36,29 @@ The following models now support metadata routing in one or more or their
 methods. Refer to the :ref:`Metadata Routing User Guide <metadata_routing>` for
 more details.
 
+- |Enhancement| :class:`~compose.ColumnTransformer` now supports metadata routing
+  according to :ref:`metadata routing user guide <metadata_routing>`. :pr:`27005`
+  by `Adrin Jalali`_.
+
+- |Enhancement| :class:`linear_model.LogisticRegressionCV` now supports
+  metadata routing. :meth:`linear_model.LogisticRegressionCV.fit` now
+  accepts ``**params`` which are passed to the underlying splitter and
+  scorer. :meth:`linear_model.LogisticRegressionCV.score` now accepts
+  ``**score_params`` which are passed to the underlying scorer.
+  :pr:`26525` by :user:`Omar Salman <OmarManzoor>`.
+
+- |Enhancement| :class:`multiclass.OneVsRestClassifier`,
+  :class:`multiclass.OneVsOneClassifier` and
+  :class:`multiclass.OutputCodeClassifier` now support metadata routing.
+  :meth:`multiclass.OneVsRestClassifier.fit`,
+  :meth:`multiclass.OneVsOneClassifier.fit` and
+  :meth:`multiclass.OutputCodeClassifier.fit` now accept ``**fit_params`` which
+  are passed to the estimator used.
+  :meth:`multiclass.OneVsRestClassifier.partial_fit` and
+  :meth:`multiclass.OneVsOneClassifier.partial_fit` now accept
+  ``**partial_fit_params`` which are passed to the estimator used. :pr:`27308`
+  by :user:`Stefanie Senger <StefanieSenger>`.
```

Comment:
We're using the `|Feature|` tag here, and this can be shortened a bit:

```
- |Feature| :class:`multiclass.OneVsRestClassifier`,
  :class:`multiclass.OneVsOneClassifier` and
  :class:`multiclass.OutputCodeClassifier` now now support metadata routing in
  their ``fit`` and ``partial_fit``, and route metadata to the underlying
  estimator's ``fit``. :pr:`27308` by :user:`Stefanie Senger <StefanieSenger>`.
```